sql/plan/show_indexes.go

```diff
@@ -64,7 +64,10 @@ func (n *ShowIndexes) Schema() sql.Schema {
 func (n *ShowIndexes) RowIter(ctx *sql.Context, row sql.Row) (sql.RowIter, error) {
 	table, ok := n.Child.(*ResolvedTable)
 	if !ok {
-		panic(fmt.Sprintf("unexpected type %T", n.Child))
+		table, ok = n.Child.Children()[0].(*ResolvedTable)
```

Comment:
This is awfully fragile. Check out getResolvedTable in the analyzer package (and maybe move it to the plan package while you're at it)

Reply:
this was a workaround for the exchange node problems, it wasn't supposed to land here